bcda/models/postgres/repository_test.go

```diff
@@ -482,7 +482,7 @@ func (r *RepositoryTestSuite) TestACOMethods() {
 		r.repository.UpdateACO(ctx, uuid.Parse(aco.ClientID),
 			map[string]interface{}{"termination_details": &models.Termination{}}),
 		fmt.Sprintf("ACO %s not updated, no row found", aco.ClientID))
-	assert.Contains(r.repository.CreateACO(ctx, aco).Error(), "duplicate key value violates unique constraint \"acos_cms_id_key\"")
+	assert.Contains(r.repository.CreateACO(ctx, aco).Error(), "duplicate key value violates unique constraint \"acos_pkey\"")
```

Comment:
NOTE: the ACOS table has 2 constraints; a unique primary key and unique cmsid.
the initial migration, 000001, created the former constraint before the latter,
thus violations with both will report the former error.  However with
migration 000015, altering the column deletes, then recreates the
unique cmsid, thus placing it second.  This test case, that violates
both constraints, is now going to report an issue with the primary key
as that is first in the list of table constraints